### DISCUSSION TITLE
Orchard Core “umlaut” (German Character) Problem

### DISCUSSION THREAD

**Discussion by pinhan34** [OrchardCMS/OrchardCore#9418] [General]
0

I have a Setup.recipe.json file in which there exists a content item with a html string containing German "umlaut" characters such as "ö" , "ü" etc. I tried adding several settings to the recipe file and i also enabled all "Internationalization" modules and i also added this code piece to my Startup.cs file:

services.Configure<WebEncoderOptions>(options =>
            {
                options.TextEncoderSettings = new TextEncoderSettings(UnicodeRanges.All);
            });

Despite all these i can not display these characters within the content after saving and publishing it. Any help would be appreciated. Thanks in advance.

**Comment by hishamco** [comment]
> Despite all these i can not display these characters within the content after saving and publishing it. 

Just to make it clear, is this mean they show up before edit? Please show us how your final output is look like, it would be nice if we could have a screenshot for the view source especially for the paragraph that contains "umlaut"
